src/components/ProfileDetailsActionMenu/ProfileDetailsActionMenu.js

```diff
@@ -21,12 +22,16 @@ export const ProfileDetailsActionMenu = ({
     onToggle();
   };
 
+  const stripes = useStripes();
+  const hasViewPerms = stripes.hasPerm('settings.data-export.view') && !stripes.hasPerm('module.data-export.enabled');
```

Comment:
Does it sound good to rename it to "hasViewOnlyPerms"? Same comment for rest of the settings pages.

Reply:
It's sounds better. Thank's. Fixed